.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,94 @@
+                    -   php: 7.1
+                        dependency: lowest
+                        symfony: '^4.0'
+                    -   php: 7.2
+                        dependency: lowest
+                        symfony: '^5.0'
+                    -   php: 8.0
+                        dependency: lowest
+                        symfony: '^6.0'
+
+                    # All supported php versions with the highest dependencies for any supported Symfony version
+                    -   php: 5.5
+                        dependency: highest
+                        symfony: 'any'
+                    -   php: 5.6
+                        dependency: highest
+                        symfony: 'any'
+                    -   php: 7.0
+                        dependency: highest
+                        symfony: 'any'
+                    -   php: 7.1
+                        dependency: highest
+                        symfony: 'any'
+                    -   php: 7.2
+                        dependency: highest
+                        symfony: 'any'
+                    -   php: 7.3
+                        dependency: highest
+                        symfony: 'any'
+                    -   php: 7.4
+                        dependency: highest
+                        symfony: 'any'
+                    -   php: 8.0
+                        dependency: highest
+                        calculate-code-coverage: yes
+                        symfony: 'any'
+
+        steps:
+            -   name: Checkout source
+                uses: actions/checkout@v2
+
+            -   name: Get code coverage driver
+                uses: actions/github-script@v3.1
+                id: code-coverage-driver
+                with:
+                    script: 'return "${{ matrix.calculate-code-coverage }}" == "yes" ? "pcov" : "none"'
+                    result-encoding: string
+
+            -   name: PHP setup
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: ${{ steps.code-coverage-driver.outputs.result }}
+
+            -   name: Require Symfony Flex
+                if: ${{ matrix.php }} >= ${{ 7.0 }}
+                run: composer global require --no-interaction --no-progress symfony/flex:^1.0.21
+
+            -   name: Require Symfony version
+                if: matrix.symfony != 'any'
+                run: composer config extra.symfony.require ${{ matrix.symfony }}
+
+            -   name: Update composer dependencies
+                run: composer update --optimize-autoloader --no-interaction --no-progress ${{ matrix.composer-flags }}
+
+            -   name: Run test suite
+                run: ./vendor/bin/phpunit -v
+
+            -   name: Upload coverage results to Coveralls
+                if: matrix.calculate-code-coverage == 'yes'
+                env:
+                    COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
                    COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
```

Reply:
Is this really needed? currently all works fine

Reply:
Here example with `secrets.GITHUB_TOKEN` https://github.com/marketplace/actions/coveralls-github-action
Or this is outdated?

Reply:
one cannot declare `secrets.GITHUB_TOKEN` anymore - https://github.com/php-coveralls/php-coveralls/issues/326

Reply:
Done